using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -4,8 +4,14 @@
 
 Retrieves the Analytics tracking identifier that is generated for this app/device instance. This identifier is an app-specific, unique visitor ID that is generated at the initial launch and is stored and used after the initial launch. The ID is preserved between app upgrades and is removed when the app is uninstalled.
 
+{% hint style="info" %}
+The value returned by `getTrackingIdentifier` may be null if you have a [Experience Cloud ID](../mobile-core/identity/identity-api-reference#get-experience-cloud-ids) and do not have visitor ID grace period configured. 
```

Comment:
Please reword so that this sentence reads as follows:

"If you have a [Experience Cloud ID](../mobile-core/identity/identity-api-reference#get-experience-cloud-ids), and do not have visitor ID grace period configured, the value returned by `getTrackingIdentifier` might be null."